src/visu_config.xsd

```diff
@@ -563,7 +564,12 @@
             <xsd:appinfo>level:expert</xsd:appinfo>
         </xsd:annotation>
     </xsd:attribute>
-    <xsd:attribute name="visible" type="xsd:boolean" />
+    <xsd:attribute name="visible" type="xsd:boolean">
+    	<xsd:annotation>
+        	<xsd:documentation xml:lang="en">Displays the widget (page name) which links to be page</xsd:documentation>
+        	<xsd:documentation xml:lang="de">Zeigt ein Widget an (Seitenname), das auf die Seite verlinkt</xsd:documentation>
```

Comment:
According to the above suggestion: "Zeigt ein Widget an, das auf die Seite verlinkt"